### PR TITLE
Feature/ockovanienh

### PR DESCRIPTION
**Pull request by msusicky**
Added environment variable ODL_VACCINATED_ENH
If filled then this path will be used as a source for the enhanced vaccination data

### CODE REVIEW

File: app/opendata_fetcher.py [hunk 1]
```diff
@@ -35,6 +36,12 @@ def fetch_all(self):
             self._fetch_used()
             self._fetch_distributed()
             self._fetch_vaccinated()
+            # Test if we have an scraped dataset or not
+            if os.environ.get('ODL_VACCINATED_ENH') is not None and os.path.exists(
```
Comment (jan-stanek) [inline app/opendata_fetcher.py:40]: os.environ.get('ODL_VACCINATED_ENH') can be stored to some variable

File: app/opendata_fetcher.py [hunk 2]
```diff
@@ -202,6 +209,82 @@ def _fetch_vaccinated(self):
...
+        df['indikace_pedagog'] = df['indikace_pedagog'].astype('bool')
+        df['indikace_skolstvi_ostatni'] = df['indikace_skolstvi_ostatni'].astype('bool')
+
+        db.session.query(OckovaniLideEnh).delete()
+
+        for row in df.itertuples(index=False):
+            db.session.add(OckovaniLideEnh(
+                datum=row[0],
+                vakcina=row[1],
+                kraj_nuts_kod=row[2],
```
Comment (jan-stanek) [inline app/opendata_fetcher.py:236]: names of columns are more readable then numbers

File: app/opendata_fetcher.py [hunk 3]
```diff
@@ -202,6 +209,82 @@ def _fetch_vaccinated(self):
...
+                zarizeni_kod=format(int(row[3]), '011d'),
```
Comment (jan-stanek) [inline app/opendata_fetcher.py:237]: this can cause issues, some codes are higher than INT_MAX

Comment (jan-stanek) [reply app/opendata_fetcher.py:237]: we read all columns as strings, so it shouldn't be needed

**Review by jan-stanek — CHANGES_REQUESTED** [review]
see comments

**Comment by msusicky** [comment]
See changes :)

File: app/opendata_fetcher.py [hunk 4]
```diff
@@ -22,9 +23,12 @@ class OpenDataFetcher:
     _check_dates = None
     _import = None
     _last_modified = None
+    _vaccinated_profese = None
 
-    def __init__(self, check_dates = True):
+    def __init__(self, check_dates=True):
         self._check_dates = check_dates
+        if os.environ.get('ODL_VACCINATED_ENH') is not None:
+            OpenDataFetcher._vaccinated_profese = os.environ.get('ODL_VACCINATED_ENH')
```
Comment (jan-stanek) [inline app/opendata_fetcher.py:31]: self

File: app/opendata_fetcher.py [hunk 5]
```diff
@@ -36,14 +40,13 @@ def fetch_all(self):
...
-            if os.environ.get('ODL_VACCINATED_ENH') is not None and os.path.exists(
-                    os.environ.get('ODL_VACCINATED_ENH')):
-                self._fetch_vaccinated_enh_path(os.environ.get('ODL_VACCINATED_ENH'))
-            else:
-                self._fetch_vaccinated_enh()
             self._fetch_registrations()
             self._fetch_reservations()
+            # Test if we have an scraped dataset or not
+            if os.path.exists(self._vaccinated_profese):
+                self._fetch_vaccinated_profese_path(self._vaccinated_profese)
```
Comment (jan-stanek) [inline app/opendata_fetcher.py:47]: vaccinated_professions would be better :)

**Review by jan-stanek — CHANGES_REQUESTED** [review]

**Review by jan-stanek — APPROVED** [review]

**Pull request merged**